packages/react-components/src/components/VideoGallery.tsx

```diff
@@ -201,7 +201,7 @@ export const VideoGallery = (props: VideoGalleryProps): JSX.Element => {
   ]);
 
   if (shouldFloatLocalVideo()) {
-    const floatingTileHostId = 'UILibaryFloatingTileHost';
+    const floatingTileHostId = 'UILibraryFloatingTileHost';
```

Comment:
what are these ids for?

Reply:
They are referenced by the layerprop hostid: https://developer.microsoft.com/en-us/fluentui#/controls/web/layer#ILayerProps

For fluent modal dialogs they essentially say, render the element in the element with this ID and not in the default location (which is at the end of the DOM directly under body)